lib/connection.c

```diff
@@ -173,6 +173,10 @@ cJSON *STDCALL create_auth_json_body(SF_CONNECT *sf,
     if (sf->password && *(sf->password)) {
         snowflake_cJSON_AddStringToObject(data, "PASSWORD", sf->password);
     }
+
+    if (sf->oauth_token && *(sf->oauth_token)) {
```

Comment:
Here we need to check authenticator type. "TOKEN" in login request has different meaning with different authenticators.
https://github.com/snowflakedb/snowflake-odbc/blob/master/Source/Platform/Connection.cpp#L605

Reply:
Done.